apps/server/src/modules/health/domain/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './health-statuses.do';
+export * from './health-status.do';
+export * from './health-status-check.do';
+export * from './healthcheck.do';
```

Comment:
```suggestion
export * from './health-check.do';
```

Reply:
Done in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4191/commits/9f7cb45820e06668af950204e05f82983c112e54 .